addons/event/models/event_mail.py

```diff
@@ -91,7 +91,9 @@ def _compute_done(self):
             if mail.interval_type in ['before_event', 'after_event']:
                 mail.done = mail.mail_sent
             else:
-                mail.done = len(mail.mail_registration_ids) == len(mail.event_id.registration_ids) and all(mail.mail_sent for mail in mail.mail_registration_ids)
+                mail.done = len(mail.mail_registration_ids) and (
+                    len(mail.mail_registration_ids) == len(mail.event_id.registration_ids) and all(mail.mail_sent for mail in mail.mail_registration_ids)
+                )
```

Comment:
```suggestion
                mail.done = len(mail.mail_registration_ids) and (
                    len(mail.mail_registration_ids) == len(mail.event_id.registration_ids) and 
                    all(mail.mail_sent for mail in mail.mail_registration_ids)
                )
```
just to ease reading